R/adorn_percentages.R

```diff
@@ -91,9 +91,9 @@ adorn_percentages <- function(dat, denominator = "row", na.rm = TRUE, ...) {
       if ("col" %in% attr(dat, "totals") & !explicitly_exempt_totals) {
         cols_to_tally <- c(cols_to_tally, ncol(dat))
         if ("row" %in% attr(dat, "totals")) {
-          col_sum <- c(col_sum, sum(dplyr::last(dat)[-nrow(dat)]))
+          col_sum <- c(col_sum, sum(dat[-nrow(dat), ncol(dat)]))
```

Comment:
Just FYI, another way to extract the last column with dplyr is `dplyr::pull(dat)`.